code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -442,8 +537,9 @@ type StorageAllocationStats struct {
 }
 
 type BlobberAllocation struct {
-	BlobberID       string                  `json:"blobber_id"`
-	AllocationID    string                  `json:"allocation_id"`
+	BlobberID    string `json:"blobber_id"`
+	AllocationID string `json:"allocation_id"`
+	// Size is blobber allocation maximum size
 	Size            int64                   `json:"size"`
```

Comment:
If it means the maximum size, we'd better rename it to `Cap` or `MaxSize`, etc. Maybe change it in the future